src/app.js

```diff
@@ -10,7 +10,7 @@ import express from 'express';
  */
 
 import indexRouter from './routes/index';
-import officeRoute from './routes/offices';
+import officeRoute from './routes/users';
```

Comment:
app.js should only import one route object, ie indexRoute. Which should be exported from the routes/index folder and should have all other routes in the object